### PR TITLE
fix: Don't publish test files

### PR DESCRIPTION
**Pull request by ffflorian**
## Pull Request Checklist

- [ ] ~~My code is covered by tests~~
- [ ] ~~I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes~~

### CODE REVIEW

File: packages/store-engine-bro-fs/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   "scripts": {
     "build": "yarn build:node",
     "build:node": "tsc",
-    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js}\" \"!src/test/**/*\"",
+    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js}\"",
```
Comment (ffflorian) [inline packages/store-engine-bro-fs/package.json:36]: There is no `src/test` dir

File: packages/store-engine-dexie/package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
   "scripts": {
     "build": "yarn build:node",
     "build:node": "tsc",
-    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js}\" \"!src/test/**/*\"",
+    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js}\"",
```
Comment (ffflorian) [inline packages/store-engine-dexie/package.json:36]: There is no `src/test` dir

File: packages/store-engine-fs/package.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "build": "yarn build:node",
     "build:node": "tsc",
-    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js}\" \"!src/test/**/*\"",
+    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js}\"",
```
Comment (ffflorian) [inline packages/store-engine-fs/package.json:31]: There is no `src/test` dir

File: packages/store-engine-sqleet/package.json [hunk 4]
```diff
@@ -44,7 +44,7 @@
   "scripts": {
     "build": "yarn build:node",
     "build:node": "tsc",
-    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js,websql-worker.js}\" \"!src/test/**/*\"",
+    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js,websql-worker.js}\"",
```
Comment (ffflorian) [inline packages/store-engine-sqleet/package.json:47]: There is no `src/test` dir

File: packages/store-engine-web-storage/package.json [hunk 5]
```diff
@@ -27,7 +28,7 @@
   "scripts": {
     "build": "yarn build:node",
     "build:node": "tsc",
-    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js}\" \"!src/test/**/*\"",
+    "clean": "del-cli \"src/**/*{.js,.js.map,.d.ts}\" \"!src/**/*{.test.node.js,.test.js}\"",
```
Comment (ffflorian) [inline packages/store-engine-web-storage/package.json:31]: There is no `src/test` dir

**Comment by bennycode** [comment]
@ffflorian nice to know: If you are maintaining a library on npm and you would like to increase the quality indicator of your library, then you would have to publish test files (checkout "Test Score" in [Increasing an NPM package’s search score](https://itnext.io/increasing-an-npm-packages-search-score-fb557f859300)). 

Since we don't care of npm's quality index of our libraries, it's okay to skip those files.

**Review by bennycode — APPROVED** [review]

**Comment by bennycode** [comment]
@ffflorian we had the revert your PR as it caused publishing packages (i.e. `@wireapp/bot-api`) which don't include source files. Here is an example of this faulty behavior: https://unpkg.com/browse/@wireapp/bot-api@7.11.53/

**Comment by ffflorian** [comment]
@bennycode this happened already before my PR was merged.

**Comment by bennycode** [comment]
Ok, I see the issue. Your PR removed the test files and because there was already a bug before (which removes publishing business logic) there was no more code at all after merging your PR.

Last working version (with source code) is @wireapp/bot-api@7.11.21. Bug appeared in @wireapp/bot-api@7.11.22.

**Comment by ffflorian** [comment]
Exactly, see e.g. https://github.com/ffflorian/wire-bots/pull/927

**Pull request merged**